sourcegraph/sourcegraph
# web: fix pnpm-lock issue

## Context

There's an issue on main caused by the presence of the `sourcegraph: ''` entry in the `pnpm-lock.yaml`. If we install any new dependency, the `pnpm install` command fails on CI. [Failure example](https://github.com/sourcegraph/sourcegraph/actions/runs/4121774771/jobs/7117792725).

## Test plan

1. CI
2. Add any dependency locally to the root package.json file and verify that `sourcegraph: ''` is not present in the lock file.
3. `bazel build //client/...` is green locally.
4. Ensure that circular references between tsconfig files do not exist — they break bazel builds.
